data/rule_data.yml

```diff
@@ -120,6 +120,8 @@ rule_data:
   - ecosystem-cert-preflight-checks
   - fips-operator-bundle-check
   - fips-operator-bundle-check-oci-ta
+  - coverity-availability-check
+  - coverity-availability-check-oci-ta
```

Comment:
I'm fairly sure that `coverity-availability-check-oci-ta` is bogus and should be deleted. Adding it here might make people think they need to use it.

Reply:
Either way, not a blocker for merging.

Reply:
I took a look - it seems to be deprecated now, which implies it's not bogus. Lgtm.